### PR TITLE
Featured channels

### LINKED ISSUE #529: Discovery: add component for featured channels
This feature will add a new component at the top of the Discovery page that contains featured channels. Those are specific channels we want to highlight. For example when we want to highlight specific channels. These could be a new additions. This list of channels should be synced when the user is online, so that we have control of it and can update it accordingly.

Image

### PR DESCRIPTION
**Pull request by manuq**
Fixes #529

### CODE REVIEW

**Review by jprvita — CHANGES_REQUESTED** [review]
After the rebase, the "Your channels" row appears empty to me (and I followed the instructions on https://github.com/endlessm/kolibri-explore-plugin/issues/529#issuecomment-1555106897 an extra time to double check).

File: kolibri_explore_plugin/assets/src/views/DiscoveryPageContenPacks.vue [hunk 1]
```diff
@@ -8,6 +8,25 @@
...
+          <b-container>
+            <h5 class="mt-2 text-muted">
+              {{ $tr('channelLabel') }}
+            </h5>
+          </b-container>
+          <SlidableGrid
+            v-slot="slotProps"
+            :nodes="sectionNodes['featured-channel']"
+          >
+            <ChannelCard
```
Comment (jprvita) [inline kolibri_explore_plugin/assets/src/views/DiscoveryPageContenPacks.vue:21]: I think this is now called `EkChannelCard`.

File: kolibri_explore_plugin/assets/src/views/DiscoveryPageContenPacks.vue [hunk 2]
```diff
@@ -8,6 +8,25 @@
     </template>
     <template v-else>
       <b-container class="mb-2 mt-4 no-container-padding">
+        <template v-if="hasNodesForSection('featured-channel')">
+          <b-container>
+            <h5 class="mt-2 text-muted">
+              {{ $tr('channelLabel') }}
+            </h5>
+          </b-container>
+          <SlidableGrid
```
Comment (jprvita) [inline kolibri_explore_plugin/assets/src/views/DiscoveryPageContenPacks.vue:17]: `EkSlidableGrid`

**Review by jprvita — CHANGES_REQUESTED** [review]

File: kolibri_explore_plugin/assets/src/views/DiscoveryPageContenPacks.vue (hunk 1 above)
Comment (manuq) [reply kolibri_explore_plugin/assets/src/views/DiscoveryPageContenPacks.vue]: Ouch! I failed to rebase this, sorry. Fixed now.

File: kolibri_explore_plugin/assets/src/views/DiscoveryPageContenPacks.vue (hunk 2 above)
Comment (manuq) [reply kolibri_explore_plugin/assets/src/views/DiscoveryPageContenPacks.vue]: Same wrong rebase issue, fixed.

**Review by jprvita — APPROVED** [review]
Looks good now!

**Pull request merged**